### PR TITLE
Updated the jitsi link to use href instead of onclick

### LINKED ISSUE #89: jitsi:// urls in the main page can't be just copied
They least to the main index, so can't really be copy pasted

### PR DESCRIPTION
**Pull request by soichih**
So that the user can copy the link target. I *believe* this closes #89

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -223,13 +223,13 @@ async function directory(jQuery) {
                     if(row.videochat && row.videochat.startsWith("http")) {
                         //or use custom url
                         return `
-                            <a href="${row.videochat}" targer="ohbm2020_${row.number}">
+                            <a href="${row.videochat}" targer="ohbm2020_${row.number}" target="room_${row.number}">
```
Comment (yarikoptic) [inline assets/js/main.js]: so multiple targets is ok and handled consistently by the browsers (e.g taking the last one)? upon a quick google, didn't information

Comment (soichih) [reply assets/js/main.js]: Oops.. good catch. I completely missed that.

Comment (soichih) [reply assets/js/main.js]: Updated the PR

File: backend/server.js [hunk 2]
```diff
@@ -82,7 +82,7 @@ setInterval(()=>{
     for(let id in ips) {
         let recents = [];
         let recent = new Date();
-        recent.setTime(recent.getTime() - 1000*60*120);  
+        recent.setTime(recent.getTime() - 1000*60*3);
```
Comment (yarikoptic) [inline backend/server.js:85]: Those is unrelated to the PR purpose, but I guess it's ok

**Review by yarikoptic — APPROVED** [review]
Will test on the morning but I guess it should work ;-) thank you @soichih

**Pull request merged**